napari/main.py

```diff
@@ -0,0 +1,40 @@
+"""
+napari command line viewer.
+"""
+import argparse
+import numpy as np
+from skimage import io
```

Comment:
do we want to use `imageio` here instead? we currently depend on both libraries so could go either way